implementations/rs/tests/it/main.rs

```diff
@@ -0,0 +1,80 @@
+#![feature(assert_matches)]
+use polywrap_client::client::PolywrapClient;
+use polywrap_core::{
+    client::ClientConfig,
+    resolvers::static_resolver::{StaticResolver, StaticResolverLike},
+    uri::Uri,
+};
+use polywrap_ethereum_wallet_plugin::{
+    connection::Connection, connections::Connections, EthereumWalletPlugin,
+};
+use polywrap_msgpack::msgpack;
+use polywrap_plugin::package::PluginPackage;
+use std::{collections::HashMap, sync::Arc};
+
+pub mod request;
+
+fn get_client() -> PolywrapClient {
+    let connection = Connection::new(
+        "https://bsc-dataseed1.binance.org/".to_string(),
+        Some(String::from(
+            "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d",
```

Comment:
Just out of curiosity, how does ethers-rs implement support for external private key stores?

Reply:
they use their [`ethers-signer` crate](https://github.com/gakonst/ethers-rs/tree/master/ethers-signers) to interact with user-related interactions; we're using their [`LocalWallet` type](https://github.com/gakonst/ethers-rs/blob/master/ethers-signers/src/lib.rs#L12) to [convert this local private key to an instance of this type](https://github.com/polywrap/ethereum-wallet/blob/df42c1687842d7cc2f1f840f9e538a679be9f8cd/implementations/rs/src/connection.rs#L78) that can be used to sign.